pkgs/applications/networking/mailreaders/claws-mail/default.nix

```diff
@@ -1,7 +1,7 @@
-{ fetchurl, stdenv, wrapGAppsHook
+{ fetchurl, stdenv, wrapGAppsHook, autoreconfHook
 , curl, dbus, dbus-glib, enchant, gtk2, gnutls, gnupg, gpgme, hicolor-icon-theme
 , libarchive, libcanberra-gtk2, libetpan, libnotify, libsoup, libxml2, networkmanager
-, openldap , perl, pkgconfig, poppler, python, shared-mime-info, webkitgtk24x-gtk2
+, openldap, perl, pkgconfig, poppler, python2Packages, shared-mime-info, webkitgtk24x-gtk2
```

Comment:
`python2Packages` is the same as `python.pkgs`, please use `python` for Python and `python.pkgs` instead of `python2Packages`.

Reply:
done